gui/window_character_finder.gui

```diff
@@ -0,0 +1,178 @@
+### LIST TEMPLATE
```

Comment:
Add this file to the master and merge to this branch so I can see what you changed.

Reply:
Done in a3ce38f